LICENSE

```diff
@@ -0,0 +1,22 @@
+MIT License
+
+Copyright (c) 2018-present, Gridsome
+Copyright (c) 2020-present, Henk Verlinde
```

Comment:
Where is this license file coming from? This repo should probably be licensed similarly to the main one, with copyright assigned to "Aegisub Project"

Reply:
The file comes from the [doks repo](https://github.com/h-enk/doks/blob/master/LICENSE) and should be changed.